### DISCUSSION TITLE
A strange problem in 'The Schmid Factor'?

### DISCUSSION THREAD

**Discussion by shaochuyou** [mtex-toolbox/mtex#2088] [Ask Anything]
I am a novice, sorry for my hard learning results. I am trying to calculate the in  'Schmid factor' ,'https://mtex-toolbox.github.io/SchmidtFactor.html'
 I am using this code (I am working with MTEX5.9.0),Here's a little problem:
Command in picture 1
SF = sS.SchmidFactor
Command in picture 2
tau = sSAll.SchmidFactor(r)
Allow me to ask a very childish question,and why is there no 'r' in front and 'r' in back (red mark)?
![1](https://github.com/mtex-toolbox/mtex/assets/84962655/2f5cd7e6-2e85-4db8-8c51-6d5081ec6899)
![2](https://github.com/mtex-toolbox/mtex/assets/84962655/54d1a971-1d34-47aa-8c42-bc197529bf08)

**Comment by Tijmenvermeij** [comment]
Hi,

It's not exactly clear to me what you mean. But I'll try:
the first command gives you the SF in all directions, so no direction r is required as input.
the second command gives you the schmid factors for multiple slip systems for a single direction, hence specifying r.

Best,
Tijmen
